dojson/utils.py

```diff
@@ -45,11 +45,11 @@ def wrapper(self, key, value, **kwargs):
 
 
 def filter_values(f):
-    """Remove None values from dictionary."""
+    """Remove None or empty values from dictionary."""
     @functools.wraps(f)
     def wrapper(*args, **kwargs):
         out = f(*args, **kwargs)
-        return dict((k, v) for k, v in six.iteritems(out) if v is not None)
+        return dict((k, v) for k, v in six.iteritems(out) if v or v is False or v == 0)
```

Comment:
The original intention of this decorator was to filter `None` values. You can extend this decorator to accept an optional argument with values that should be filtered.

```
@filter_values(values=(None, False, ''))
def foo(*bar, **baz):
    pass
```

IMHO the condition is not safe since it's impossible to list all objects that evaluate to `False` and should be included.